apps/vh/base/nonprod-role.yaml

```diff
@@ -0,0 +1,43 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nonprod-team-permissions
+rules:
+- apiGroups: ['']
+  resources: ['pods']
+  verbs: ['delete']
+- apiGroups: ['']
+  resources: ['secrets']
+  verbs: ['*']
+- apiGroups: ['helm.fluxcd.io']
```

Comment:
don't we need a new api group for flux 2?

ref https://hmcts.github.io/ways-of-working/troubleshooting/#latest-image-is-not-updated-in-cluster